<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="lineEdit_link">
        <property name="toolTip">
         <string>link1</string>
        </property>
        <property name="statusTip">
         <string>link2</string>
        </property>
        <property name="whatsThis">
         <string>link3</string>
        </property>
        <property name="accessibleName">
         <string>link4</string>
        </property>
        <property name="accessibleDescription">
         <string>link5</string>
        </property>
        <property name="inputMask">
         <string/>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Normal</enum>
        </property>
        <property name="placeholderText">
         <string>لینک را وارد کنید و بجای عددی که باید تغییر کند *** بزارید</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="prefix">
         <string>Start </string>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_num">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;تعداد لینک هایی که می خواهید تولید شود&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;برای تغییر دادن عدد می توانید از Wheel موس استفاده کنید&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="statusTip">
         <string>تعداد لینک هایی که می خواهید تولید شود</string>
        </property>
        <property name="whatsThis">
         <string>تعداد لینک هایی که می خواهید تولید شود</string>
        </property>
        <property name="wrapping">
         <bool>false</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::UpDownArrows</enum>
        </property>
        <property name="suffix">
         <string> Part</string>
        </property>
        <property name="prefix">
         <string/>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>2</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit_resualt">
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="documentTitle">
       <string/>
      </property>
      <property name="lineWrapColumnOrWidth">
       <number>18</number>
      </property>
      <property name="overwriteMode">
       <bool>false</bool>
      </property>
      <property name="tabStopWidth">
       <number>80</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushBtn_generate">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ساخت لینک</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionFile">
   <property name="text">
    <string>File</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit_link</tabstop>
  <tabstop>spinBox_num</tabstop>
  <tabstop>pushBtn_generate</tabstop>
  <tabstop>textEdit_resualt</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
